config/airsim_octo.rviz

```diff
@@ -152,9 +185,11 @@ Visualization Manager:
         map:
           base_link:
             base_link/Lidar2:
-              {}
+              velodyne:
```

Comment:
Possibly out of scope but maybe the references to 'velodyne' should be abstracted to a generic LIDAR name/format

Reply:
Yes I do want to do that, but we'll need to coordinate with Francisco, it's a hardcoded header frame_ID in rs_to_velodyne